### ISSUE TITLE
Keyboard failure during new chroot install (trusty)

### ISSUE THREAD

**Issue by Pshriwise** [dnschneid/crouton#2255]
Whether using an in browser chrosh or VT-2 crosh, when installing a new trusty chroot (with no or any targets) the keyboard fails to work on anything but VT-2. Works after rebooting the machine, but its impossible to properly finish the install if not using VT-2. 

My chromebook is an Acer cp720.

**Comment by Sunappnio** [comment]
I found that the easiest way is to use a usb mouse, and then open a screen keyboard

**Comment by patleeman** [comment]
I had the same issue, had to plug in a usb keyboard to complete installation.

**Comment by darcyparker** [comment]
Seems #1994 is the first issue where this was reported.

**Comment by SkylabInc** [comment]
I tried to install crouton with unity-trusty today. And I face the issue of keyboard not responding at the end of the installation when it asks chroot username. Restarted the chrome pixel and then went to crosh's shell and entered sudo enter-chroot command. And now it asked the username and password again. And that made me login successfully. I searched every where on web for the solution. My trail and error solved this. But not sure if there will be any side effects as this is first time I am using chrome book
